worldedit_commands/init.lua

```diff
@@ -456,15 +456,35 @@ minetest.register_chatcommand("/replaceinverse", {
 })
 
 local check_sphere = function(name, param)
-	if worldedit.pos1[name] == nil then
-		worldedit.player_notify(name, "no position 1 selected")
-		return nil
-	end
 	local found, _, radius, nodename = param:find("^(%d+)%s+(.+)$")
 	if found == nil then
 		worldedit.player_notify(name, "invalid usage: " .. param)
 		return nil
 	end
+	if nil ~= area_protection.areas then

```

Comment:
might be better to move this into a `area_protection.active()` function (returns boolean)